### PR TITLE
Galacto linting

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by deltakosh** [comment]
@PolygonalSun please pay extra attention to the input classes. I move some PAI from protected to public

**Comment by deltakosh** [comment]
Thanks to the tool and the fact I lost my both eyes I found some gems like this
image

**Comment by deltakosh** [comment]
THE MOTHER OF ALL PR!!!

**Comment by deltakosh** [comment]
@CedricGuillemet / @syntheticmagus can you please get it locally and make sure I did not break anything for Native

@PolygonalSun can you do the same and do an extensive test for inputs?

**Review by RaananW — APPROVED** [review]

File: packages/dev/core/src/Cameras/Inputs/arcRotateCameraPointersInput.ts [hunk 1]
```diff
@@ -231,24 +230,23 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
...
         this._isPanClick = evt.button === this.camera._panningMouseButton;
     }
 
     /**
      * Called each time a new POINTERUP event occurs. Ie, for each button
      * release.
-     * @param evt
      */
-    protected onButtonUp(evt: IPointerEvent): void {
+    public onButtonUp(): void {
```
Comment (PolygonalSun) [inline packages/dev/core/src/Cameras/Inputs/arcRotateCameraPointersInput.ts:241]: This is supposed to override what's in BaseCameraPointersInput so I feel like this should match what is in that class, either by re-adding `evt` or removing it from the BaseCameraPointersInput class version.

**Review by PolygonalSun — COMMENTED** [review]
All of the DSM/DIS stuff looks fine to me.  I just have one thing that stands out to me.

**Comment by PolygonalSun** [comment]
> 

I'll pull the repo and run it through the same tests that I do up to BRN changes.

**Comment by PolygonalSun** [comment]
> @CedricGuillemet / @syntheticmagus can you please get it locally and make sure I did not break anything for Native
> 
> @PolygonalSun can you do the same and do an extensive test for inputs?

@deltakosh, I'll pull the repo and run it through the same tests that I do up to BRN changes.

**Comment by PolygonalSun** [comment]
> > @CedricGuillemet / @syntheticmagus can you please get it locally and make sure I did not break anything for Native
> > @PolygonalSun can you do the same and do an extensive test for inputs?
> 
> @deltakosh, I'll pull the repo and run it through the same tests that I do up to BRN changes.

This works fine for the JS side, I'm currently testing Native side now.

**Comment by PolygonalSun** [comment]
> > > @CedricGuillemet / @syntheticmagus can you please get it locally and make sure I did not break anything for Native
> > > @PolygonalSun can you do the same and do an extensive test for inputs?
> > 
> > 
> > @deltakosh, I'll pull the repo and run it through the same tests that I do up to BRN changes.
> 
> This works fine for the JS side, I'm currently testing Native side now.

Native side works.

**Comment by deltakosh** [comment]
Thanks a million Dave!

**Pull request merged**